docs/src/tutorials/huggingface.rst

```diff
@@ -0,0 +1,183 @@
+*****************************************************************************************
+Hyperparameters optimisation using a HuggingFace Model and the Hydra-Orion-Sweeper plugin
+*****************************************************************************************
+
+In this tutorial, we will show an easy Orion integration of a HuggingFace translation model using
+Hydra, with the `Hydra_Orion_Sweeper <https://github.com/Epistimio/hydra_orion_sweeper>`_ plugin.
+Hydra is essentially a framework for configuring applications. We will use it to define our
+Hyperparameters and some Orion configuration. We will also be using
+`Comet <https://www.comet.com/>`_ for monitoring our experiments.
+
+Imports
+^^^^^^^
+You will need to import some modules using pip, such as ``datasets``, ``transformers``,
+``hydra-orion-sweeper`` and ``comet-ml``.
+
+.. literalinclude:: /../../examples/huggingface/main.py
+  :language: python
+  :lines: 5-6,13-22
+
+
+Hydra configuration file
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+Notice here how the arguments that are not defined will be set at ``None``, which will be
+overridden by default values or not used at all. This serves as a replacement for parsing arguments
+in the command line, but is integrated with Orion, which makes the usability really good.
```

Comment:
This part is not totally clear to me. What do you mean by _making the usability really good_? Do you refer to the fact that we can define the arguments in the yaml file instead of cmdline? Or that we can define more HPs than the one used by Oríon, just setting then to None when we don't want to optimize them?

Reply:
Ok, maybe saying something like _which makes it more practical than cmdline arguments to manage search spaces of hyperparameters_ would be more clear.